pkg/builder/client/client.go

```diff
@@ -86,3 +92,25 @@ func (c *client) Build(ctx context.Context, req *builder.PackageBuildRequest) (*
 
 	return &pkgBuildResp, ferror.MakeErrorFromHTTP(resp)
 }
+
+func (c *client) Clean(ctx context.Context, req *builder.PackageCleanRequest) error {
+	logger := otelUtils.LoggerWithTraceID(ctx, c.logger)
+
+	body, err := json.Marshal(req)
+	if err != nil {
+		return errors.Wrap(err, "error marshaling json")
+	}
+
+	resp, err := ctxhttp.Post(ctx, c.httpClient.StandardClient(), c.getCleanUrl(), "application/json", bytes.NewReader(body))
+	if err != nil {
+		logger.Error("error creating delete request", zap.Error(err))
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
```

Comment:
We should be able to ignore not found error etc if API is missing due to older version builder on function side.

Reply:
Done